### ISSUE TITLE
Custom formmater does not work witj NX 18 and stylelint 16

### ISSUE THREAD

**Issue by ugrave** [Phillip9587/nx-stylelint#634]
if a use a external formatter (in this example [stylelint-checkstyle-reporter](https://github.com/lucavb/stylelint-checkstyle-reporter)), the formatter is not used and the output is stylelint json.
The output should be the output of the formatter (which is in this case xml)

target definition in nx.json: 
```json
    "stylelint": {
      "inputs": ["default", "{workspaceRoot}/.stylelintrc.json", "{workspaceRoot}/.stylelintignore"],
      "cache": true,
      "options": {
        "formatter": "stylelint-checkstyle-reporter"
      }
    }
```
When i run stylelint via cli, the formatter is working:
`npx stylelint "*.scss" --custom-formatter=stylelint-checkstyle-reporter`

Versions:
nx: 18.0.2
stylelint: 16.2.1
nx-stylelint: 17.1.2
stylelint-checkstyle-reporter: 1.0.0

**Comment by ugrave** [comment]
I found a workaround:
I created a seperated script file which exports a function which calls internally the real for-matter function

stylelint-checkstyle-reporter-fix.js:
```javascript
const { stylelintToCheckstyle } = require('stylelint-checkstyle-reporter');

module.exports = function (results) {
  return stylelintToCheckstyle(results);
}
```

Use the fix formatter in nx.json:
nx.json:
```json
    "stylelint": {
      "inputs": ["default", "{workspaceRoot}/.stylelintrc.json", "{workspaceRoot}/.stylelintignore"],
      "cache": true,
      "options": {
        "formatter": "./stylelint-checkstyle-reporter-fix.js"
      }
    }
```

**Comment by Phillip9587** [comment]
@ugrave I'm looking into it. I already fixed the deprecation warning in [9c02cda ](https://github.com/Phillip9587/nx-stylelint/commit/9c02cda239e2513df4a29b205dc757a2424f6519).

**Comment by Phillip9587** [comment]
should be fixed in [v17.1.3](https://github.com/Phillip9587/nx-stylelint/releases/tag/v17.1.3)
